### PR TITLE
fix(core): explicitly disable admin rights for Windows binaries

### PR DESCRIPTION
**Pull request by merceyz**
**What's the problem this PR addresses?**

`binjumper` doesn't explicitly disable the need for admin rights to spawn it so when its name contains "install", "setup", "update", or "patch" Windows thinks it's an installer and requires administrator rights to start the binary.

**How did you fix it?**

Updated `binjumper` to get the fix from upstream https://github.com/merceyz/binjumper/commit/bb271872f45fb74bfe7e54e65c1f5e8a298261a0

**Checklist**
- [x] I have read the [Contributing Guide](https://yarnpkg.com/advanced/contributing).
- [x] I have set the packages that need to be released for my changes to be effective.
- [x] I will check that all automated PR checks pass before the PR gets reviewed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,17 @@
 
 ### CLI
 
+- Explicitly disable administrator rights for the native binary jumper on Windows. When a filename contains "install", "setup", "update", or "patch" Windows thinks it's an installer and requires administrator rights to start the binary
+
+## 2.3.1
+
+### CLI
+- Take into account peer dependency defaults when inheriting workspace peer dependencies in the node_modules linker
```
Comment (merceyz) [inline CHANGELOG.md:14]: @larixer More or less copied your commit message

Comment (larixer) [reply CHANGELOG.md:14]: Its fine, thank you!

Comment (larixer) [reply CHANGELOG.md:14]: It reads like someone is drilling through your head with hammerdrill, exactly the impression I was after

Comment (arcanis) [reply CHANGELOG.md:14]: For changelogs, it's fine if you stay vague and high-level: "Fixes an hoisting problem that could happen when workspaces listed a package as both a peer dependency and a regular one". People reading those usually don't care about the implementation, they just want to know if they are/will be impacted by the change.

**Pull request merged**